docs/tutorials/running_on_a_server.md

```diff
@@ -14,9 +14,9 @@ We assume the SLURM cluster using the _enroot_ container software, which is a re
 
 With enroot installed, you can follow the below steps to run OmniGibson on SLURM: 
 
-1. Download the dataset to a location that is accessible by cluster nodes. To do this, you can use the download_dataset.py script inside OmniGibson's scripts directory, and move it to the right spot later. In the below example, /cvgl/ is a networked drive that is accessible by the cluster nodes. **For Stanford users, this step is already done for SVL and Viscam nodes**
+1. Download the dataset to a location that is accessible by cluster nodes. To do this, you can use the download_datasets.py script inside OmniGibson's scripts directory, and move it to the right spot later. In the below example, /cvgl/ is a networked drive that is accessible by the cluster nodes. **For Stanford users, this step is already done for SVL and Viscam nodes**
 ```{.shell .annotate}
-OMNIGIBSON_NO_OMNIVERSE=1 python scripts/download_dataset.py
+OMNIGIBSON_NO_OMNIVERSE=1 python omnigibson/download_datasets.py
 mv omnigibson/data /cvgl/group/Gibson/og-data-0-2-1
```

Comment:
maybe this ID needs to be updated? @cgokmen

Reply:
I am updating these but I think it's safe to let them fall behind later. People should hopefully get the gist.